README.md

```diff
@@ -1,9 +1,50 @@
+
+## Getting Started
+The instructions below would get you a copy of the project up and running on your local machine
+
+### Prerequisite
+- Git
+- Node
+- npm or yarn
+
+### Installation
+- Clone the repository `git clone https://github.com/TREP-LABS/realdrip-backend.git`
+- Change into the directory of the project
+- Use `npm install` to install all the dependency packages
+- Create a `.env` file in the root folder of the to provide all the needed environment variables as specified in `.env.example`
+- Use `npm run dev` to start the application in dev mode
+
+## Continuous Integration(CI)
+We try our best to continuously integrate our changes together as soon as they are created i.e we don't do long live feature branches. All pull requests are raised directly unto the shared main branch(master). You can read more about continuous integrations [here](https://en.wikipedia.org/wiki/Continuous_integration). We use a third-party platform to effectively manage our continuous integration, [TravisCI](https://travis-ci.org/).Travis helps us run automated builds for every commit introduced into the codebase. At any point in time, if Travis reports a failing build, fixing such a build should be the priority of the team. For CI, Travis builds includes linting the codebase to ensure it adheres to the standard of the team and running automated tests to ensure nothing is breaking.
+
+## Continuous Delivery(CD)
+We also practice Continuous Delivery, every change introduced(merged) into the master branch is automatically deployed to a staging(pre-production) environment where it can be tested as it would be in production. You can read more about Continuous Delivery[here](https://en.wikipedia.org/wiki/Continuous_delivery);
+This practice makes us production-ready always, given that everything works well in this staging environment, we can always go to production without any hassle. We also make use of Travis for our Continuous Delivery, however, because we can't afford to deploy every single change that is pushed, we only auto deploy changes merged to the master branch.
+
+## Folder Structure
+- The root folder mostly contains config and meta files.
+- The src folder is where the actual code lives.
+- `src/http`: This folder contains all the Http and Http related logic used in the application. Everything from instantiating and configuring the express app, routing, request validation, etc. is in this folder.
+- `src/service`: This folder contains all the business logic of the application. The logic of how we go about creating or authenticating a user, updating an infusion, etc. can be found in this folder. The approach of separating actual business logic from controller Http code might be a little foreign, however, we did this to avoid issues like having a bloated controller, read more about that [here](https://www.coreycleary.me/why-should-you-separate-controllers-from-services-in-node-rest-apis/).
```

Comment:
```suggestion
- `src/service`: This folder contains all the business logic of the application. The logic of creating or authenticating a user, updating an infusion, etc. can be found in this folder. The approach of separating actual business logic from controller Http code might be foreign, however, we did this to avoid issues like having a bloated controller. Read more about separating controllers from services [here](https://www.coreycleary.me/why-should-you-separate-controllers-from-services-in-node-rest-apis/).
```